### ISSUE TITLE
force:source:deploy feedback wonky

### ISSUE THREAD

**Issue by julesx** [forcedotcom/cli#622]
### Summary

```
C:\...\classes>sfdx force:source:deploy -p "c:\...\classes\Test_Rem_TargetList.cls"
Job ID | 0Af3h00000DRnHACA1
SOURCE PROGRESS | ████████████████████████████████████████ | 1/1 Components
=== Deployed Source
FULL NAME            TYPE       PROJECT PATH
───────────────────  ─────────  ────────────
Test_Rem_TargetList  ApexClass
Test_Rem_TargetList  ApexClass

```

```
C:\...\classes>sfdx update
sfdx-cli: Updating CLI from 7.73.2-feef087df8 to 7.74.1-32db2396ed... done
sfdx-cli: Updating CLI... done
sfdx-cli: Updating plugins... done
```

```
C:\...\classes>sfdx force:source:deploy -p "c:\...\classes\Test_Rem_TargetList.cls"
=== Deployed Source
No results found
```

### Steps To Reproduce:

Update sfdx
Deploy file

### Expected result

user feedback should describe what happened

### Actual result

I received a cryptic message

**OS and version**: Win 7 Pro

**Comment by shetzel** [comment]
Is the targeted class within a packageDirectory defined within sfdx-project.json?  I'm guessing it's not.

**Comment by julesx** [comment]
```
{
  "packageDirectories": [
    {
      "path": "code\\AEM\\sfdc",
      "default": true
    }
  ],
  "namespace": "",
  "sfdcLoginUrl": "https://login.salesforce.com",
  "sourceApiVersion": "48.0"
}
```

`code\AEM\sfdc` is one of the directories between `C:\` and `classes`

**Comment by shetzel** [comment]
In the top screenshot the deploy results list nothing in the `PROJECT PATH` column, which means it didn't match the deploy result to a file within your project.  That used to work (arguably, it shouldn't when it can't find the matching file in the project) but with the recent update to the `source:deploy` and `source:retrieve` commands to support multiple package directories (mpd) it stopped working.  We'll be fixing that asap.  However, in your case the file _IS_ in the project but it's not finding it, which is a slightly different bug.

As a workaround for now, can you try to play with the paths to get it working.  E.g., try some of these small changes:
1. `sfdx force:source:deploy -p "code\AEM\sfdc\classes\Test_Rem_TargetList.cls"`
2. `sfdx force:source:deploy -p ".\code\AEM\sfdc\classes\Test_Rem_TargetList.cls"`
3. execute the command from the project root rather than in the code\AEM\sfdc\classes dir
4. change packageDirectory path to `.\\code\\AEM\\sfdc`

Please let me know if any of those changes affect the deploy result and output.

**Comment by shivani-ar** [comment]
Is this fixed? I've been having similar issue. Not able to deploy anything in Destination org(Sandbox).  For example - Trying to deploy permission set/apex class/Layout or anything. It is coming with following errors:

1. The PermissionSet named BDR_Extra_Access was not found in the workspace.
2. The ApexClass named Accounts was not found in the workspace.

**Comment by mshanemc** [comment]
closed 9/22/2020 cf W-8100862. @shivani-ar if you can still reproduce this, please open a new issue
